lstchain/irf/hdu_table.py

```diff
@@ -0,0 +1,354 @@
+
+            hdu_tables.append(t_aeff)
+        except:
+            log.error('Effective Area HDU not found')
+        ###############################################
+        #Background
+        try:
+            Table.read(filepath, hdu="BACKGROUND")
+            t_bkg = t_edisp.copy()
+            t_bkg['HDU_TYPE'] = ['bkg']
+            t_bkg['HDU_CLASS'] = ['bkg_2d']
+            t_bkg['HDU_CLASS2'] = ['BKG']
+            t_bkg['HDU_CLASS4'] = ['BKG_2D']
+            t_bkg['HDU_NAME'] = ['BACKGROUND']
+
+            hdu_tables.append(t_bkg)
+        except:
+            log.error('Background HDU not found')
+        ###############################################
+        # Obs_table
+        t_obs = Table(
+            {'OBS_ID' : [event_table.meta['OBS_ID']],
+            'DATE_OBS' : [event_table.meta['DATE_OBS']],
+            'RA_PNT' : [pointing_table.meta['RA_PNT']],
+            'DEC_PNT' : [pointing_table.meta['DEC_PNT']],
+            'ZEN_PNT' : [90 - float(pointing_table.meta['ALT_PNT'])],
+            'ALT_PNT' : [pointing_table.meta['ALT_PNT']],
+            'AZ_PNT' : [pointing_table.meta['AZ_PNT']],
+            'RA_OBJ' : [event_table.meta['RA_OBJ']],
+            'DEC_OBJ' : [event_table.meta['DEC_OBJ']],
+            'ONTIME': [event_table.meta["ONTIME"]],
+            'LIVETIME' : [event_table.meta["LIVETIME"]],
+            'DEADC' : [event_table.meta["DEADC"]],
+            'TSTART' : [event_table.meta["TSTART"]],
+            'TSTOP' : [event_table.meta["TSTOP"]],
+            'OBJECT' : [event_table.meta['OBJECT']],
+            'OBS_MODE' : [event_table.meta['OBS_MODE']],
+            'N_TELS' : [event_table.meta["N_TELS"]],
+            'TELLIST' : [event_table.meta["TELLIST"]]},
+            dtype=('>i8', '>S12','>f4', '>f4', '>f4', '>f4', '>f4',
+            '>f4', '>f4','>f4', '>f4', '>f4', '>f8', '>f8', 'S20',
+            '>S20', '>i8', 'S20')
+            )
+        obs_tables.append(t_obs)
+
+    hdu_table = vstack(hdu_tables)
+
+    hdu_header = DEFAULT_HEADER.copy()
+    hdu_header["HDUCLAS1"] = "INDEX"
+    hdu_header["HDUCLAS2"] = "HDU"
+    hdu_header["TELESCOP"] = "CTA"
+    hdu_header["INSTRUME"] = "LST-1"
+
+    filename_hdu_table = fits_dir/'hdu-index.fits.gz'
+
+    hdu = fits.BinTableHDU(hdu_table, header=hdu_header, name='HDU INDEX')
+    hdu_list = fits.HDUList([fits.PrimaryHDU(), hdu])
+    hdu_list.writeto(filename_hdu_table, overwrite=True)
+
+    obs_table = vstack(obs_tables)
+
+    obs_header = hdu_header.copy()
+    obs_header["HDUCLAS2"] = "OBS"
+    obs_header['MJDREFI'] = event_table.meta['MJDREFI']
+    obs_header['MJDREFF'] = event_table.meta['MJDREFF']
+
+    filename_obs_table = fits_dir/'obs-index.fits.gz'
+
+    obs = fits.BinTableHDU(obs_table, header = obs_header, name='OBS INDEX')
+    hdu_list = fits.HDUList([fits.PrimaryHDU(), obs])
+    hdu_list.writeto(filename_obs_table, overwrite=True)
+
+    return
+
+def create_event_list(data, run_number, source_name):
```

Comment:
A lot of hard coded things.
Might be necessary right now, but we should keep track of those.

Reply:
Indeed!